src/core/control/tools/BaseStrokeHandler.h

```diff
@@ -47,7 +47,6 @@ class BaseStrokeHandler: public InputHandler {
 private:
     virtual void drawShape(Point& currentPoint, const PositionInputData& pos) = 0;
     DIRSET_MODIFIERS drawModifierFixed = NONE;
-    int lastCursor = -1;  // avoid same setCursor
```

Comment:
This will disappear on its own when #4137 gets merged (presumably by the end of the week).